packages/core/src/common/_color-aliases.scss

```diff
@@ -7,7 +7,8 @@
 Color aliases
 
 These variables are semantic aliases of our [colors](#colors). They are used throughout Blueprint itself to ensure
-consistent color usage across components.
+consistent color usage across components. Override these variables to change the overall look and feel for your
+application.
```

Comment:
mm this sentence is more confusing than helpful. don't think it's necessary yet, please remove. we'll draw up more complete docs on sass consumption after this feature merges.